test/commute.js

```diff
@@ -17,7 +16,26 @@ describe('commute', function() {
   });
 
   it('works on Algebraic Data Types such as "Maybe"', function() {
```

Comment:
> Note that you didn't change the name, `Maybe` -> `Identity`.

I considered doing so, but it didn't seem appropriate. I was reading _such as "Maybe"_ as a hint to readers as to what an algebraic data type is. Given this interpretation, _such as "Identity"_ would be less useful as I imagine there are fewer people familiar with Identity than with Maybe.

> But are there compelling reasons to not allow a test to require an implementation of `Maybe` […]?

It makes it clear which methods the ADT must provide. `R.commute`, for example, requires `ap` and `map`.

I'm not *strongly* opposed to adding a Maybe implementation as a dependency.

> I also personally prefer the semantics of `Maybe`, as people actually understand that one for the most part, and recognize that it, in fact, does something useful. Using `Identity` feels as though it doesn't really demonstrate anything.

The example previously on line 20 just involved Just. I thought since we were only showing the behaviour of Just we might as well give the container a more straightforward name. What we should do instead, though, is test the behaviour of Nothing as well as Just. I'll update this.